kmip/pie/client.py

```diff
@@ -542,6 +544,65 @@ def activate(self, uid=None):
             message = result.result_message.value
             raise exceptions.KmipOperationFailure(status, reason, message)
 
+    def revoke(self, revocation_reason, uid=None, revocation_message=None,
+               compromise_occurrence_date=None):
+        """
+        Revoke a managed object stored by a KMIP appliance.
+
+        Args:
+            revocation_reason (RevocationReasonCode): An enumeration indicating
+                the revocation reason.
+            uid (string): The unique ID of the managed object to revoke.
+                Optional, defaults to None.
+            revocation_message (string): A message regarding the revocation.
+                Optional, defaults to None.
+            compromise_occurrence_date (int): A integer which will be converted
```

Comment:
Let's be a little more specific here:
"An integer, the number of seconds since the epoch, which will be converted..."